FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc

```diff
@@ -378,18 +377,17 @@ bool CTPPSFastTrackingProducer::SearchTrack(int i,
     return false;
   //
   // Calculate the reconstructed track parameters
-  theta = sqrt(thx * thx + thy * thy) * urad;
+  double theta = sqrt(thx * thx + thy * thy) * urad;
   xi = eloss / fBeamEnergy;
   double energy = fBeamEnergy * (1. - xi);
   partP = sqrt(energy * energy - PPSTools::ProtonMassSQ);
   t = -2. * (PPSTools::ProtonMassSQ - fBeamEnergy * energy + fBeamMomentum * partP * cos(theta));
-  pt = sqrt(pow(partP * thx * urad, 2) + pow(partP * thy * urad, 2));
+  pt = partP * theta;
```

Comment:
@sandrofonseca this lines reproduces what was coded before
Could you please confirm that the approximation `sin(theta) == theta` was really wanted?

Reply:
> @sandrofonseca this lines reproduces what was coded before
> Could you please confirm that the approximation `sin(theta) == theta` was really wanted?

@jan-kaspar @fabferro FYI

Reply:
For forward protons the scattering angles are not larger than O(10^-4 rad), thus the approximation is well justified.

Reply:
> For forward protons the scattering angles are not larger than O(10^-4 rad), thus the approximation is well justified.

Thank you Jan: I just wanted to get confirmation that I did not inadvertently hit a typo with this update.

Reply:
Thank you for double checking!